contrib/completion/zsh/_docker-compose

```diff
@@ -29,7 +29,7 @@ __docker-compose_all_services_in_compose_file() {
     local -a services
     already_selected=$(echo $words | tr " " "|")
     __docker-compose_q config --services \
-        | grep -Ev "$already_selected"
```

Comment:
I could join cbfe6d0 and 7e80ab4.

However, I'm not sure this is a good idea. They address two separate issues, which just happened to be on the same line (originally), so I think it's a good idea to leave them as two separate commits.

Especially if I'll be splitting de216f3 up into smaller commits.